heavyai/_pandas_loaders.py

```diff
@@ -246,6 +253,20 @@ def _serialize_arrow_payload(data, table_metadata, preserve_index=True):
         else:
             data_ = data
 
+        # convert geo columns to WKT representation
+        for col in table_metadata:
+            if col.type in GEO_TYPE_NAMES:
+                try:
+                    fn = functools.partial(shapely.to_wkt,
+                                           rounding_precision=-1,
+                                           trim=False,
+                                           output_dimension=2)
+                    data_[col.name] = data_[col.name].apply(fn)
+                except TypeError:
```

Comment:
What is the typical TypeError exception message here? Could we verify that the message is as expected before transforming TypeError to ValueError?

Reply:
The `TypeError` message would be something along the lines: 

```
TypeError: One of the arguments is of incorrect type. Please provide only Geometry objects.
```